Manager/Doctrine/ClientManager.php

```diff
@@ -34,4 +35,51 @@ public function save(Client $client): void
         $this->entityManager->persist($client);
         $this->entityManager->flush();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function remove(Client $client): void
+    {
+        $this->entityManager->remove($client);
+        $this->entityManager->flush();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function list(?ClientFilter $clientFilter): array
+    {
+        $repository = $this->entityManager->getRepository(Client::class);
+
+        if (!$clientFilter || !$clientFilter->hasFilters()) {
+            return $repository->findAll();
```

Comment:
@Allypost Check if you can simplify your flow just by calling the `findBy` method with an empty array.